recipes/microlens-submit/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set setup = load_setup_py_data() %}
+{% set python_min = "3.8" %}
+
+package:
+  name: microlens-submit
+  version: {{ setup.get("version") }}
+
+source:
+  url: https://files.pythonhosted.org/packages/source/m/microlens-submit/microlens_submit-0.16.3.tar.gz
```

Comment:
I would suggest using the canonical url of: `https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz`

and then set the version as shown in the example `meta.yaml`:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L9-L13